app/Platform/Jobs/UpdatePlayerPointsStatsBatchJob.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Jobs;
+
+use App\Platform\Actions\UpdatePlayerPointsStatsBatchAction;
+use Illuminate\Bus\Batchable;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Carbon;
+
+class UpdatePlayerPointsStatsBatchJob implements ShouldQueue
+{
+    use Batchable;
+    use Dispatchable;
+    use InteractsWithQueue;
+    use Queueable;
+    use SerializesModels;
+
+    /**
+     * @param array<int> $userIds
+     */
+    public function __construct(
+        private readonly array $userIds,
+        private readonly ?string $mockCurrentDate = null, // for testing only
```

Comment:
If this just results in a call to `Carbon::setTestNow`, can't you just do that in the test instead of making it a parameter to the job?